applications/GeoMechanicsApplication/tests/cpp_tests/test_U_Pw_condition.cpp

```diff
@@ -0,0 +1,52 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+
+// Project includes
+#include "custom_conditions/U_Pw_condition.hpp"
+
+#include "geo_mechanics_fast_suite.h"
+
+using namespace Kratos;
+namespace Kratos::Testing {
```

Comment:
This file still needs to be formatted with clang-format. In visual studio there is an extension for using clang-format and if you set it up correctly (such that it uses the `applications/GeoMechanicsApplication/.clang-format), it will automatically format it in the same way as the rest of the geo code base.